.pipelines/.vsts-vhd-windows-automation.yaml

```diff
@@ -0,0 +1,48 @@
+pool:
+  # uncomment this if the auto teardown pool is down.
+  # vmImage: ubuntu-22.04
+  name: $(1ES_DevInfra_Auto_TearDown_Pool_Name)
+
+parameters:
+- name: ImageBump
+  displayName: Image Bump + Branch Cutting
+  type: boolean
+  default: true
+- name: ReleaseNotes
+  displayName: Release Notes
+  type: boolean
+  default: true
+- name: ArtifactTrigger
+  displayName: SIG Artifact Trigger
+  type: boolean
+  default: true
+
+steps:
+- bash: |
```

Comment:
Do we need to configure go?

Reply:
It's needed for release notes later for calling `generate_release_notes` in `automate_release_notes.sh`. Removed now.